### PR TITLE
MSTR-160 : 채점 가능 및 활성화 여부 수정 필드 추가

### PR DESCRIPTION
**Pull request by kshired**
### Issue Number

close: MSTR-160

### 작업 내역

채점 가능 및 활성화 여부 수정 필드 추가

### 변경사항

N/A

### 작업 유형

- [x] 신규 기능 추가
- [ ] 버그 수정
- [ ] 리펙토링
- [ ] 문서 업데이트

### PR 특이 사항

- isGradable, isActive를 수정 가능하도록 변경하였습니다.
- isGradable과 isActive가 단건 조회시 response되도록 수정하였습니다.
- 그에 따른 docs와 test code 수정이 있었습니다.

### CODE REVIEW

File: src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt [hunk 1]
```diff
@@ -208,7 +208,11 @@ class AdminControllerTest {
                         PayloadDocumentation.fieldWithPath("gradingStandards.[].score")
                             .type(JsonFieldType.NUMBER).description("채점기준 점수"),
                         PayloadDocumentation.fieldWithPath("gradingStandards.[].type")
-                            .type(JsonFieldType.STRING).description("채점기준 타입 ( 'KEYWORD' or 'PROMPT' )")
+                            .type(JsonFieldType.STRING).description("채점기준 타입 ( 'KEYWORD' or 'PROMPT' )"),
+                        PayloadDocumentation.fieldWithPath("isGradable")
+                            .type(JsonFieldType.BOOLEAN).description("채점 가능 여부 ( 필수 x, 기본 값 false )"),
+                        PayloadDocumentation.fieldWithPath("isActive")
+                            .type(JsonFieldType.BOOLEAN).description("활성화 여부 ( 필수 x, 기본 값 true )"),
```
Comment (kshired) [inline src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt:215]: 🚫 **[ktlint]** <trailing-comma> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary trailing comma before ")"

File: src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt [hunk 2]
```diff
@@ -357,7 +361,11 @@ class AdminControllerTest {
                         PayloadDocumentation.fieldWithPath("gradingStandards.[].score")
                             .type(JsonFieldType.NUMBER).description("채점기준 점수"),
                         PayloadDocumentation.fieldWithPath("gradingStandards.[].type")
-                            .type(JsonFieldType.STRING).description("채점기준 타입 ( 'KEYWORD' or 'PROMPT' )")
+                            .type(JsonFieldType.STRING).description("채점기준 타입 ( 'KEYWORD' or 'PROMPT' )"),
+                        PayloadDocumentation.fieldWithPath("isGradable")
+                            .type(JsonFieldType.BOOLEAN).description("채점 가능 여부"),
+                        PayloadDocumentation.fieldWithPath("isActive")
+                            .type(JsonFieldType.BOOLEAN).description("활성화 여부"),
```
Comment (kshired) [inline src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt:368]: 🚫 **[ktlint]** <trailing-comma> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary trailing comma before ")"

File: src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt [hunk 3]
```diff
@@ -461,7 +469,11 @@ class AdminControllerTest {
                         PayloadDocumentation.fieldWithPath("data.gradingStandards.[].score")
                             .type(JsonFieldType.NUMBER).description("채점기준 점수"),
                         PayloadDocumentation.fieldWithPath("data.gradingStandards.[].type")
-                            .type(JsonFieldType.STRING).description("채점기준 타입 ( 'KEYWORD' or 'PROMPT' )")
+                            .type(JsonFieldType.STRING).description("채점기준 타입 ( 'KEYWORD' or 'PROMPT' )"),
+                        PayloadDocumentation.fieldWithPath("data.isGradable")
+                            .type(JsonFieldType.BOOLEAN).description("채점 가능 여부"),
+                        PayloadDocumentation.fieldWithPath("data.isActive")
+                            .type(JsonFieldType.BOOLEAN).description("활성화 여부"),
```
Comment (kshired) [inline src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt:476]: 🚫 **[ktlint]** <trailing-comma> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary trailing comma before ")"

File: src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt [hunk 4]
```diff
@@ -508,7 +520,11 @@ class AdminControllerTest {
                         PayloadDocumentation.fieldWithPath("answer").type(JsonFieldType.STRING)
                             .description("정답"),
                         PayloadDocumentation.fieldWithPath("score")
-                            .type(JsonFieldType.NUMBER).description("채점기준 점수")
+                            .type(JsonFieldType.NUMBER).description("채점기준 점수"),
+                        PayloadDocumentation.fieldWithPath("isGradable")
+                            .type(JsonFieldType.BOOLEAN).description("채점 가능 여부 ( 필수 x, 기본 값 true )"),
+                        PayloadDocumentation.fieldWithPath("isActive")
+                            .type(JsonFieldType.BOOLEAN).description("활성화 여부 ( 필수 x, 기본 값 true )"),
```
Comment (kshired) [inline src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt:527]: 🚫 **[ktlint]** <trailing-comma> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary trailing comma before ")"

File: src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt [hunk 5]
```diff
@@ -566,7 +582,11 @@ class AdminControllerTest {
                         PayloadDocumentation.fieldWithPath("answer").type(JsonFieldType.STRING)
                             .description("문제 정답"),
                         PayloadDocumentation.fieldWithPath("score")
-                            .type(JsonFieldType.NUMBER).description("문제 점수")
+                            .type(JsonFieldType.NUMBER).description("문제 점수"),
+                        PayloadDocumentation.fieldWithPath("isGradable")
+                            .type(JsonFieldType.BOOLEAN).description("채점 가능 여부"),
+                        PayloadDocumentation.fieldWithPath("isActive")
+                            .type(JsonFieldType.BOOLEAN).description("활성화 여부"),
```
Comment (kshired) [inline src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt:589]: 🚫 **[ktlint]** <trailing-comma> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary trailing comma before ")"

File: src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt [hunk 6]
```diff
@@ -626,6 +646,10 @@ class AdminControllerTest {
                             .description("문제 정답"),
                         PayloadDocumentation.fieldWithPath("data.score").type(JsonFieldType.NUMBER)
                             .description("문제 점수"),
+                        PayloadDocumentation.fieldWithPath("data.isGradable")
+                            .type(JsonFieldType.BOOLEAN).description("채점 가능 여부"),
+                        PayloadDocumentation.fieldWithPath("data.isActive")
+                            .type(JsonFieldType.BOOLEAN).description("활성화 여부"),
```
Comment (kshired) [inline src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt:652]: 🚫 **[ktlint]** <trailing-comma> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary trailing comma before ")"

File: src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt [hunk 7]
```diff
@@ -685,7 +709,11 @@ class AdminControllerTest {
                         PayloadDocumentation.fieldWithPath("choices.[].isAnswer").type(JsonFieldType.BOOLEAN)
                             .description("선지 정답 여부"),
                         PayloadDocumentation.fieldWithPath("score")
-                            .type(JsonFieldType.NUMBER).description("채점기준 점수")
+                            .type(JsonFieldType.NUMBER).description("채점기준 점수"),
+                        PayloadDocumentation.fieldWithPath("isGradable")
+                            .type(JsonFieldType.BOOLEAN).description("채점 가능 여부 ( 필수 x, 기본 값 true )"),
+                        PayloadDocumentation.fieldWithPath("isActive")
+                            .type(JsonFieldType.BOOLEAN).description("활성화 여부 ( 필수 x, 기본 값 true )"),
```
Comment (kshired) [inline src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt:716]: 🚫 **[ktlint]** <trailing-comma> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary trailing comma before ")"

File: src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt [hunk 8]
```diff
@@ -759,7 +787,11 @@ class AdminControllerTest {
                         PayloadDocumentation.fieldWithPath("choices.[].isAnswer").type(JsonFieldType.BOOLEAN)
                             .description("선지 정답 여부"),
                         PayloadDocumentation.fieldWithPath("score")
-                            .type(JsonFieldType.NUMBER).description("채점기준 점수")
+                            .type(JsonFieldType.NUMBER).description("채점기준 점수"),
+                        PayloadDocumentation.fieldWithPath("isGradable")
+                            .type(JsonFieldType.BOOLEAN).description("채점 가능 여부"),
+                        PayloadDocumentation.fieldWithPath("isActive")
+                            .type(JsonFieldType.BOOLEAN).description("활성화 여부"),
```
Comment (kshired) [inline src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt:794]: 🚫 **[ktlint]** <trailing-comma> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary trailing comma before ")"

File: src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt [hunk 9]
```diff
@@ -837,6 +869,10 @@ class AdminControllerTest {
                             .type(JsonFieldType.BOOLEAN).description("선지 정답 여부"),
                         PayloadDocumentation.fieldWithPath("data.score").type(JsonFieldType.NUMBER)
                             .description("문제 점수"),
+                        PayloadDocumentation.fieldWithPath("data.isGradable")
+                            .type(JsonFieldType.BOOLEAN).description("채점 가능 여부"),
+                        PayloadDocumentation.fieldWithPath("data.isActive")
+                            .type(JsonFieldType.BOOLEAN).description("활성화 여부"),
```
Comment (kshired) [inline src/test/kotlin/com/csbroker/apiserver/controller/AdminControllerTest.kt:875]: 🚫 **[ktlint]** <trailing-comma> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary trailing comma before ")"

**Comment by kshired** [comment]
🚀 Deployed on https://62f49640413ea8757ef5ae1e--csbroker-docs.netlify.app

**Review by ekzm8523 — APPROVED** [review]
Long Good Time Minute

**Pull request merged**